### PR TITLE
Wb develop

### PR DESCRIPTION
**Pull request by shakythesherpa**
This is a test to add contents to a country study page.

### CODE REVIEW

File: content/study/posts/congo-mb.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "version": 8,
+  "sources": {
+    "35kv": {
+      "tiles": [
+        "https://aep-tiles.staging.derilinx.com/tiles/ke/35kv/{z}/{x}/{y}.pbf"
```
Comment (olafveerman) [inline content/study/posts/congo-mb.json:6]: These are tiles for Kenya. When the user enables the layer, nothing will show over Congo.

File: content/study/posts/congo.yml [hunk 2]
```diff
@@ -2,6 +2,76 @@ title: Congo
...
+    category: input
+    mbLayer: 35kv
+    info: This dataset contains electricity transmission lines both existing and planned, 35 kV upwards in Congo. The dataset was provided by AICD.
+    source:
+      name: energydata.info
+      url: https://energydata.info/dataset/kenya-kenya-electricity-network
+  - id: wind
+    name: Mean wind speed
+    category: input
+    mbLayer: wind
```
Comment (olafveerman) [inline content/study/posts/congo.yml:61]: The file `content/study/posts/congo-mb.json` should contain the configuration for the `wind` layer.

File: content/study/posts/congo.yml [hunk 3]
```diff
@@ -2,6 +2,76 @@ title: Congo
...
+    type: number
+    datum: 
+      value: 3000
+      unit:
+platform:
+  title: Congo Final Report
+  url: https://worldbankgroup-my.sharepoint.com.mcas.ms/personal/rjordan_worldbank_org/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Frjordan%5Fworldbank%5Forg%2FDocuments%2FWork%2FOperations%2FAFR%20Geospatial%2FCongo%2FFinal%20Report%2FRC%2D%20Planification%20Geospatiale%20%2D%20Rapport%20final%2D%20V6%2Epdf&parent=%2Fpersonal%2Frjordan%5Fworldbank%5Forg%2FDocuments%2FWork%2FOperations%2FAFR%20Geospatial%2FCongo%2FFinal%20Report
+layers:
+  - id: 11kv
+    name: Existing Transmission Networl
```
Comment (olafveerman) [inline content/study/posts/congo.yml:51]:
```suggestion
    name: Existing Transmission Network
```

**Review by olafveerman — CHANGES_REQUESTED** [review]

**Comment by olafveerman** [comment]
@shakythesherpa Thanks for the PR! Made a couple of comments, let me know if you need additional info.

File: content/study/posts/congo-mb.json (hunk 1 above)
Comment (shakythesherpa) [reply content/study/posts/congo-mb.json:6]: Oops. I updated it with Congo dataset.

File: content/study/posts/congo.yml (hunk 2 above)
Comment (shakythesherpa) [reply content/study/posts/congo.yml:61]: It should be fixed

**Comment by shakythesherpa** [comment]
Olfa, Please see if my edits on Congo page are now ok.  Thank you.

**Comment by shakythesherpa** [comment]
@olafveerman  can you please help check this pull request?

**Review by olafveerman — APPROVED** [review]

**Comment by olafveerman** [comment]
@shakythesherpa I made some minor changes, and pushed them to a branch in this repository. Closing this PR in favor of #68

**Pull request closed without merging**